fabric/src/main/java/net/blay09/mods/balm/fabric/compat/ClothConfigUtils.java

```diff
@@ -54,9 +54,9 @@ public static ConfigScreenFactory<?> getConfigScreen(String modId) {
     private static void addPropertyToBuilder(MutableLoadedConfig config, ConfiguredProperty<?> property, ConfigCategory categoryInstance, ConfigBuilder builder) {
         final var schema = property.parentSchema();
         final var i18nBase = schema.identifier().getNamespace() + ".configuration";
-        var categoryI18nBase = i18nBase + "." + property.category();
-        var displayName = Component.translatable(categoryI18nBase + "." + property);
-        var tooltip = Component.translatable(categoryI18nBase + "." + property + ".tooltip");
+        var categoryI18nBase = i18nBase + (property.category().isEmpty() ? "" : ("." + property.category()));
```

Comment:
This is something I would *personally* change. However, this would probably be a breaking change and I don't know if this is something you would like to have...
I can also revert this, if needed/wanted!